chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java

```diff
@@ -510,6 +509,14 @@ private Surveillance convertToDomain(PendingSurveillanceEntity pr) {
 				surv.getRequirements().add(req);
 			}
 		}
+		
+		if(pr.getValidation() != null && pr.getValidation().size() > 0) {
+			for(PendingSurveillanceValidationEntity validation : pr.getValidation()) {
+				if(validation.getMessageType() == ValidationMessageType.Error) {
+					surv.getErrorMessages().add(validation.getMessage());
```

Comment:
There were two types of message types added in the data-model PR, right? Warning and Error? Should this section have something like:

    if (validation.getmessagetype == validationtype.warning)
        surv.getwarningmessages.add(validation.getmessage)
?